.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.11"]
+        python-version: ["3.12"]
```

Comment:
Since we declare supporting python >= 3.8 should we add all the versions in the matrix?

Reply:
I think using the lower and the upper bound will be good enough. See GH-72.